src/components/Filters/enums.ts

```diff
@@ -0,0 +1,23 @@
+export const patterns = {
```

Comment:
These patterns should be dropped from DS, we shouldn't export anything related to the actual usage of DS components. And the patterns should be implemented directly in the platform.

Reply:
At worst should be moved to mocks

Reply:
@youssefssc I'm not aware of use case to have predefined patterns. I would keep it as it was and define each pattern separately for particular story together with pattern message.